test/infra/control/run-tests-isolated.bash

```
@@ -394,7 +394,14 @@ docker run \
                                 # codecov-cli's network_root_folder is the runner's
                                 # /home/runner/work/proxysql/proxysql and the repo content
                                 # lives at <network_root>/proxysql/, so only paths shaped
-                                # as `SF:proxysql/...` get resolved -- the other two forms
+                                # as 'SF:proxysql/...' get resolved -- the other two forms
+                                # (note: single quotes, not backticks -- this whole bash
+                                # script is wrapped in `bash -c "..."`, and inside the
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)